database/db_pg.js

```diff
@@ -0,0 +1,10 @@
+const Pool = require("pg").Pool;
```

Comment:
Out of curiosity, why use `const Pool  = require("pg").Pool` rather than `const { Pool } = require('pg')`?